### ISSUE TITLE
Login WebDAV fails when username and password pasted

### ISSUE THREAD

**Issue by onejgordon** [OpenArchive/Save-app-android#459]
**Describe the bug**

Login fails when you paste password into the WebDAV login.

**To Reproduce**

1. Select button to log in to WebDAV private server
2. Paste URL, username, and password
3. Edit password field to remove any extraneous words pasted
4. Click sign in
5. See error saying username or password failed

**Expected behavior**

Should be possible to sign in after pasting password into the password input.

**Environment (please complete the following information):**
 - OS version: Android 13
 - Device: Pixel 7
 - App Version: `0.3.0-alpha4`

**Additional context**

Going back, then inputting username and password manually solved the problem. I am not clear whether editing the password after pasting is necessary to reproduce the bug.

**Comment by tladesignz** [comment]
Cannot reproduce. This works just fine for me.

This pretty much sounds like either the password or the username was actually really wrong. Maybe you pasted some space along with either one which shouldn't be there?

Unfortunately, we cannot do a lot to make it easier for users to spot issues with their credentials:

- Servers don't tell us, if the username is bad or the password, out of security concerns. (Attackers could enumerate usernames on a server, if the server gave away that information.)
- We cannot trim and/or remove characters since usernames and passwords can literally be *any unicode character*.

**Comment by tladesignz** [comment]
@onejgordon, do you have more detailed information? Otherwise I'll close this.

**Comment by onejgordon** [comment]
I don't have more details, unfortunately, but I can say I tried multiple times and was pasting directly from Bitwarden into the field, which I do frequently and never causes problems. There was another tester who was simultaneously experiencing a login issue, and after I noted that I fixed the problem by manually entering, it seemed to resolve their issue too.

I understand that if it can't be reproduced it can't be addressed though, so might make sense to close this anyway.
